.github/workflows/pst-amt-sock-kickoff.yml

```diff
@@ -2,13 +2,46 @@ name: SOCKS Issue creation kickoff Workflow
 on:
   issues:
     types: [opened, edited, reopened]
-    
+
 env:
   ISSUE_ID: ${{ github.event.issue.number }}
 
 jobs:
+  parse_ssh_key:
+    runs-on: ubuntu-latest
+    outputs:
+      ssh_key: ${{ fromJSON(steps.parse.outputs.payload)[' Public SSH Key'] }}
+    steps:
+      - name: Run Issue form parser
+        id: parse
+        uses: department-of-veterans-affairs/issue-forms-body-parser@main
+        with:
+          issue_id: ${{ github.event.inputs.issue_number }}
+          repository: ${{ env.GH_REPO_FOR_ISSUES }}
+          separator: '###'
+          label_marker_start: " "
+          label_marker_end: "\n"
+  validate_ssh_key:
+    runs-on: ubuntu-latest
+    needs: parse_ssh_key
+    env:
+      PUBLIC_KEY: "${{ needs.parse_ssh_key.outputs.ssh_key }}"
+    if: |
+      needs.parse_ssh_key.outputs.ssh_key != ''
+    steps:
+      - name: Run ssh public key validation
+        id: fingerprint
+        shell: bash
+        run: ssh-keygen -lf /dev/stdin <<< "$PUBLIC_KEY"
+      - name: Notify failure
+        id: ssh_key_invalid
+        shell: bash
+        run: "echo \"You seem to have uploaded an invalid ssh public key: $PUBLIC_KEY\""
```

Comment:
@flooose Where does this error surface?

Reply:
Here's a sample of what it looks like in my terminal:
```
[SOCKS Issue creation kickoff Workflow/parse_ssh_key]   ✅  Success - Main Run Issue form parser
[SOCKS Issue creation kickoff Workflow/parse_ssh_key] 🏁  Job succeeded
[SOCKS Issue creation kickoff Workflow/validate_ssh_key] 🚀  Start image=catthehacker/ubuntu:act-latest
[SOCKS Issue creation kickoff Workflow/validate_ssh_key]   🐳  docker pull image=catthehacker/ubuntu:act-latest platform=linux/amd64 username= forcePull=true
[SOCKS Issue creation kickoff Workflow/validate_ssh_key]   🐳  docker create image=catthehacker/ubuntu:act-latest platform=linux/amd64 entrypoint=["tail" "-f" "/dev/null"] cmd=[]
[SOCKS Issue creation kickoff Workflow/validate_ssh_key]   🐳  docker run image=catthehacker/ubuntu:act-latest platform=linux/amd64 entrypoint=["tail" "-f" "/dev/null"] cmd=[]
[SOCKS Issue creation kickoff Workflow/validate_ssh_key] ⭐ Run Main Run ssh public key validation
[SOCKS Issue creation kickoff Workflow/validate_ssh_key]   🐳  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/fingerprint.sh] user= workdir=
| /dev/stdin is not a public key file.
[SOCKS Issue creation kickoff Workflow/validate_ssh_key]   ❌  Failure - Main Run ssh public key validation
[SOCKS Issue creation kickoff Workflow/validate_ssh_key] exitcode '255': failure
[SOCKS Issue creation kickoff Workflow/validate_ssh_key] ⭐ Run Main Notify failure
[SOCKS Issue creation kickoff Workflow/validate_ssh_key]   🐳  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/ssh_key_invalid.sh] user= workdir=
[SOCKS Issue creation kickoff Workflow/validate_ssh_key]   ❗  ::error::You seem to have uploaded an invalid ssh public key: <-- KEY CONTENTS -->
[SOCKS Issue creation kickoff Workflow/validate_ssh_key]   ✅  Success - Main Notify failure
[SOCKS Issue creation kickoff Workflow/validate_ssh_key] 🏁  Job failed
Error: Job 'validate_ssh_key' failed
```

Reply:
That is very pretty 😄 

So I'm guessing it will be in the workflow logs? I think posting a comment to the issue would be the most visible way to notify someone that their ssh key failed. But if you want to get this merged and test it out and then make improvements later, I'm good with that.

Reply:
Let me look into that. Maybe it's easy. My assumption was that it would block other steps, but a comment might make it more prominant.

Reply:
Ok, I've got the comment in!